frontend/src/components/Checkbox/CheckboxOthers.tsx

```diff
@@ -0,0 +1,62 @@
+import { ChangeEvent, cloneElement, isValidElement, useRef } from 'react'
+import {
+  Checkbox,
+  CheckboxProps,
+  Flex,
+  forwardRef,
+  useMergeRefs,
+  useMultiStyleConfig,
+} from '@chakra-ui/react'
+
+interface CheckboxOthersProps extends CheckboxProps {
+  displayValue: string
+}
+
+export const CheckboxOthers = forwardRef<CheckboxOthersProps, 'input'>(
+  ({ children, displayValue, ...props }, ref) => {
+    const inputRef = useRef<HTMLInputElement | null>(null)
+    const mergedRef = useMergeRefs(ref, inputRef)
+
+    const handleInputChange = () => {
+      if (!inputRef?.current?.checked) {
+        inputRef?.current?.click()
+      }
+    }
+
+    return (
+      <>
+        <Checkbox {...props} ref={mergedRef}>
+          {displayValue}
+        </Checkbox>
+        <OthersWrapper onInputChange={handleInputChange}>
+          {children}
+        </OthersWrapper>
+      </>
+    )
+  },
+)
+
+const OthersWrapper = ({
+  children,
+  onInputChange,
+  onClick,
+}: {
+  children: React.ReactNode
+  onInputChange: () => void
+  onClick?: (e: ChangeEvent<HTMLInputElement>) => void
+}) => {
+  const style = useMultiStyleConfig('Checkbox', {}).others
```

Comment:
nit: prefer not assigning with `.others` so it is obvious that the style being used is `style.others`